src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -393,13 +463,11 @@ class BtreeLBAManager : public LBAManager {
     update_func_t &&f,
     LogicalCachedExtent*);
 
-  alloc_extent_ret _alloc_extent(
+  alloc_extents_ret _alloc_extents(
     Transaction &t,
     laddr_t hint,
-    extent_len_t len,
-    pladdr_t addr,
-    paddr_t actual_addr,
-    LogicalCachedExtent*);
+    extent_len_t total_len,
```

Comment:
nit, `total_len` can be calculated inside `_alloc_extents()`, which is more inclusive.